static/js/src/entry/user-portal/App.vue

```diff
@@ -1,6 +1,10 @@
 <template>
-  <error-view v-if="context.hasError" />
-  <router-view v-else />
+  <div>
+    <nav style="height:45px;background:#fff;border-bottom:1px solid #dcdcdc" />
+    <error-view v-if="context.hasError" />
+    <router-view v-else />
+    <footer style="height:300px;background:#222;" />
+  </div>
 </template>
```

Comment:
Could we use our `has-bg-white` and `has-bg-black-off` here to keep the hex values dynamic? The other inlines make sense.

Reply:
Oh lol those are completely temporary cuz I'm gonna handle the nav and footer in a different branch. Should've noted.